sqladmin/widgets.py

```diff
@@ -68,3 +69,62 @@ def __call__(self, field: Field, **kwargs: Any) -> str:
         kwargs.setdefault("data-role", "timepicker")
         kwargs.setdefault("data-date-format", "HH:mm:ss")
         return super().__call__(field, **kwargs)
+
+
+class ListWidget:
```

Comment:
Can't we use the `ListWidget from the WTForms package?

Reply:
The reason is default `ListWidget` is not suited for tabler radio style.

Reply:
Can you explain please what is the difference? Do we have to create another one? Can't we inherit and modify it?

Reply:
The default `ListWidget` use html ul or ol tag, but in tabler it looks ugly:

<img width="220" alt="image" src="https://user-images.githubusercontent.com/12986082/181273377-5c1907dc-9b3c-4804-b977-176f0ed49344.png">

For your advice, maybe can inherit from it.

Reply:
But wtforms ListWidget force tag in `<ol>` and `<ul>` and subfield tag in `<li>`:

https://github.com/wtforms/wtforms/blob/master/src/wtforms/widgets/core.py#L100

Reply:
Or we can add a custom style to change this.